cwl/src/main/scala/cwl/ParameterContext.scala

```diff
@@ -11,4 +11,7 @@ object ParameterContext {
   )
 }
 
-case class ParameterContext(inputs: WdlValue, self: WdlValue, runtime: WdlValue)
+case class ParameterContext(
+                             inputs: WdlValue = WdlOptionalValue(WdlNothingType, None),
```

Comment:
ToL: I hope we'll be able to narrow down the type a bit more than `WdlNothingType` later 😄 
`inputs`and `runtime` should be `WdlMap` I think ?

Reply:
hm I think this code is unnecessary.

Reply:
I.e. It's unnecessary given `Empty` and `withInputs`.